models/nxos/spanning_tree/nxos_spanning_tree.yml

```diff
@@ -0,0 +1,55 @@
+module: nxos_spanning_tree
+short_description: Resource module to configure spanning tree.
+description:
+  - This module configures and manages the attributes of Spanning-tree on Cisco NXOS.
+version_added: 6.2.0
+author: Vinay Mulugund (@roverflow)
+notes:
+  - Tested against Cisco IOSXE Version 17.3 on CML.
+  - This module works with connection C(network_cli).
+    See U(https://docs.ansible.com/ansible/latest/network/user_guide/platform_nxos.html)
+options:
+  config:
+    description: A dict of Spanning-tree options.
+    type: dict
+    suboptions:
+      bridge_assurance:
```

Comment:
Can we merge `bridge_assurance` and `bridge_domain` into a single key with `bridge` as the parent?

Reply:
bridge assurance are bridge domain are seprate commands like so - spanning-tree bridge assurance and spanning-tree bridge-domain 1-5 , they do not have the same prefix word bridge, do you still suggest me to add it under a parent bridge or should i keep it like this (edited)